marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -71,6 +72,15 @@ void testCreateNamespacedModel() {
         createNamespacedModel();
     }
 
+    @Test
+    @WithMockUser(roles = {"writeEntityModel"})
+    void testUpdatedIndexes() {
```

Comment:
It's not clear what this test is testing. We need to identify the scenario somewhere - i.e. that we are verifying that if a user manually removes an index that was created for a property that no longer needs an index, the index isn't accidentally restored when the entity model is saved. 

Thus, I'd expect the test method to read like this (pseudocode):

- create a model with a facetable dateTime property
- change property type to date
- verify both date and dateTime indexes exist
- manually remove dateTime index from final database
- add another property
- verify that the dateTime index wasn't added back to the staging database

Reply:
Done!!